### DISCUSSION TITLE
Adding repositorry

### DISCUSSION THREAD

**Discussion by DjangoBlok** [michaelarnauts/home-assistant-comfoconnect#25] [Q&A]
When trying to add repository https://github.com/michaelarnauts/home-assistant-comfoconnect to Home Assistant.  I get the message that it's not a valid add-on repository.
Any idea what is causing this message?

**Comment by LancevdMeer** [comment]
Have you installed HACS to Home-Assistant? 
Look here https://hacs.xyz/docs/setup/download/

The repository is already in the HACS store

HACS allows custom repostitories to be added to home-assistant
